slo/rules.go

```diff
@@ -1392,3 +1392,24 @@ func (o Objective) GenericRules() (monitoringv1.RuleGroup, error) {
 		Rules:    rules,
 	}, nil
 }
+
+func (o *Objective) genericObjectiveLabels(sloName string) map[string]string {
+	labels := o.commonRuleLabels(sloName)
+	labels["description"] = o.Description
+
+	switch o.IndicatorType() {
+	case Latency:
+		for _, matcher := range o.Indicator.Latency.Success.LabelMatchers {
+			if matcher.Name == "le" {
+				d, err := time.ParseDuration(matcher.Value + "s") // assume that the value is always in seconds
+				if err == nil {
+					labels["info"] = fmt.Sprintf("%.3f%% in %s must be faster than %s", o.Target*100, o.Window, d.String())
```

Comment:
Using `%.3f%%` formats the float using the 3 decimals. Maybe we can use `strconv.FormatFloat(o.Target * 100, 'f', -1, 64)` at the end and then use its string in the `fmt.Sprintf`.